admin/worker/billing_repair.go

```diff
@@ -0,0 +1,42 @@
+package worker
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+func (w *Worker) repairOrgBilling(ctx context.Context) error {
+	startTime := time.Now().UTC()
+	t, err := w.admin.DB.FindBillingRepairedOn(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get last billing repair time: %w", err)
+	}
+
+	ids, err := w.admin.DB.FindOrganizationIDsWithoutPaymentCreatedOnOrAfter(ctx, t)
```

Comment:
Could we just try for all orgs without a payment ID each time the job runs?

Reply:
sure that can also be done, will be simpler as well.